src/overlays/actors/ovl_En_An/z_en_an.c

```diff
@@ -13,160 +20,3441 @@
+                (otherObjectSlot > OBJECT_SLOT_NONE)) {
+                gSPSegment(POLY_OPA_DISP++, 0x06, play->objectCtx.slots[otherObjectSlot].segment);
+                gSPDisplayList(POLY_OPA_DISP++, gAnju3BroomDL);
+                gSPSegment(POLY_OPA_DISP++, 0x06, play->objectCtx.slots[originalObjectSlot].segment);
+            }
+            break;
+
+        case ENAN_ACCESSORY_CHOPSTICKS:
+            if ((this->stateFlags & ENAN_STATE_DRAW_CHOPSTICKS) && !this->forceDraw) {
+                gSPDisplayList(POLY_OPA_DISP++, gAnju1ChopsticksDL);
+            }
+            break;
+
+        case ENAN_ACCESSORY_MOON_MASK:
+            otherObjectSlot = this->msmoObjectSlot;
+            if (this->drawMoonMask && (otherObjectSlot > OBJECT_SLOT_NONE)) {
+                static Vec3f D_80B58E68 = { 450.0f, 700.0f, -760.0f };
+                static Vec3s D_80B58E74 = { 0x238C, 0, -0x3FFC };
+
+                Matrix_TranslateRotateZYX(&D_80B58E68, &D_80B58E74);
+
+                gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(play->state.gfxCtx),
+                          G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+
+                gSPSegment(POLY_OPA_DISP++, 0x06, play->objectCtx.slots[otherObjectSlot].segment);
+                gSPDisplayList(POLY_OPA_DISP++, gMoonMaskDL);
+                gSPSegment(POLY_OPA_DISP++, 0x06, play->objectCtx.slots[originalObjectSlot].segment);
+            }
+            break;
+
+        default:
+            break;
+    }
+
+    Matrix_Pop();
+
+    CLOSE_DISPS(play->state.gfxCtx);
+}
+
+s32 EnAn_ChangeCutscene(EnAn* this, s16 csId) {
+    s32 ret = false;
+
+    if (CutsceneManager_GetCurrentCsId() == CS_ID_GLOBAL_TALK) {
+        CutsceneManager_Stop(CS_ID_GLOBAL_TALK);
+        CutsceneManager_Queue(csId);
+    } else if (CutsceneManager_IsNext(csId)) {
+        CutsceneManager_StartWithPlayerCs(csId, &this->actor);
+        ret = true;
+    } else {
+        CutsceneManager_Queue(csId);
+    }
+
+    return ret;
+}
+
+s16 EnAn_GetCsId(EnAn* this, s32 numCutscenes) {
+    s16 csId = this->actor.csId;
+    s32 i;
+
+    for (i = 0; i < numCutscenes; i++) {
+        csId = CutsceneManager_GetAdditionalCsId(csId);
+    }
+
+    return csId;
+}
+
+s16 EnAn_GetChildCsId(EnAn* this, s32 numCutscenes) {
+    Actor* child = this->actor.child;
+    s16 csId = -1;
+    s32 i;
+
+    if ((child != NULL) && (child->update != NULL)) {
+        csId = child->csId;
+
+        for (i = 0; i < numCutscenes; i++) {
+            csId = CutsceneManager_GetAdditionalCsId(csId);
+        }
+    }
+
+    return csId;
+}
+
+s32 EnAn_MsgEvent_ReceiveLetterFromPostman(Actor* thisx, PlayState* play) {
+    EnAn* this = THIS;
+    s16 csId = EnAn_GetCsId(this, 0);
+    s32 ret = false;
+
+    switch (this->msgEventState) {
+        case 0x0:
+            ret = false;
+            if (!EnAn_ChangeCutscene(this, csId)) {
+                break;
+            }
+            FALLTHROUGH;
+
+        case 0x2:
```

Comment:
```suggestion
            FALLTHROUGH;
        case 0x2:
```
No spaces after fallthrough (applies 2 other places)